lib/takeaway.rb

```diff
@@ -0,0 +1,34 @@
+require_relative "../lib/basket.rb"
+
+class Takeaway
+
+  attr_reader :menu
+
+  DEFAULT_MENU = {
+    "Burger" => { name: "Burger", price: 3 },       
+    "Chips" => { name: "Chips", price: 1 },
+    "Kebab" => { name: "Kebab", price: 5 } }
+
+  def initialize(menu: DEFAULT_MENU)
+    @menu = menu
+    @basket = Basket.new
+  end
+  
+  def menu
+    string = ""
+    @menu.keys.each do |item|
```

Comment:
I think this would also work...

```ruby
@menu.each do |item|
  string += "#{item[:name]} : £#{item[:price]}\n"
 end
```
Which version do you prefer, and why?

Reply:
Ah, no, it that would require your menu to be an array of hashes, rather than a hash of hashes!